database/src/main/java/com/pip/chatbot/repository/food/FoodRepository.java

```diff
@@ -66,14 +64,14 @@
         return Optional.ofNullable(result.into(Mark.class));
     }
 
-    public Optional<Double> getAvgRestaurantMark(int id, Double zomatoMark) {
+    public Optional<Double> getAvgRestaurantMark(int id) {
         var avgMark = dsl.select(avg(MARK_RESTAURANT.MARK).as("mark"))
                 .from(MARK_RESTAURANT)
                 .where(MARK_RESTAURANT.RESTAURANT_ID.eq(id))
                 .groupBy(MARK_RESTAURANT.RESTAURANT_ID)
                 .fetchOne();
```

Comment:
Dajmy tutaj fetchOptional, zarówno dla czytelności jak i poprawności działania. Bo teraz fetchOne albo zwróci nulla albo rzuci wyjątek(w przypadku query, które nie zwróci żadnych rezultatów). Jeżeli zwróci nulla, to wyjątek poleci podczas Optional.of(), bo on nulla nie przyjmuje. W obu przypadkach skończymy wyjątkiem, a skoro zwracamy Optionala, to nie powinniśmy robić w ten sposób, bo łamiemy umowę:)